src/base/bittorrent/tracker.cpp

```diff
@@ -153,7 +153,7 @@ struct Tracker::TrackerAnnounceRequest
 {
     QHostAddress socketAddress;
     QByteArray claimedAddress;  // self claimed by peer
-    InfoHash infoHash;
+    TorrentID infoHash;
```

Comment:
Please rename the variable.

Reply:
Are you sure? It matches the name of the parameter in the request.

Reply:
From the other hand it would be better when name represents what field really means.
Changed.